frontend/test/components/scheduledpublishingDateTime.spec.ts

```diff
@@ -0,0 +1,232 @@
+      props: { isPublishable: true },
+    })
+
+    const dateField = screen.getByLabelText("Terminiertes Datum")
+    await fireEvent.update(dateField, "40.01.2050")
+
+    expect(dateField).toHaveValue("40.01.2050")
+    expect(screen.getByLabelText("Termin setzen")).toBeDisabled()
+
+    expect(
+      screen.getByTestId("scheduledPublishingDate_errors"),
+    ).toBeInTheDocument()
+    expect(screen.getByText("Kein valides Datum.")).toBeVisible()
+  })
+
+  it("should show error on incomplete date", async () => {
+    mockDocUnitStore()
+    render(ScheduledPublishingDateTime, {
+      props: { isPublishable: true },
+    })
+
+    const dateField = screen.getByLabelText("Terminiertes Datum")
+    await fireEvent.update(dateField, "40.01.205")
+    await fireEvent.blur(dateField)
+
+    expect(dateField).toHaveValue("40.01.205")
+    expect(screen.getByLabelText("Termin setzen")).toBeDisabled()
+
+    expect(
+      screen.getByTestId("scheduledPublishingDate_errors"),
+    ).toBeInTheDocument()
+    expect(screen.getByText("Unvollständiges Datum.")).toBeVisible()
+  })
+
+  it("should show error on invalid time", async () => {
+    mockDocUnitStore()
+    render(ScheduledPublishingDateTime, {
+      props: { isPublishable: true },
+    })
+
+    const timeField = screen.getByLabelText("Terminierte Uhrzeit")
+    await fireEvent.update(timeField, "invalid")
+    await fireEvent.blur(timeField)
+
+    expect(timeField).toHaveValue("")
+    expect(screen.getByLabelText("Termin setzen")).toBeDisabled()
+
+    expect(
+      screen.getByTestId("scheduledPublishingDate_errors"),
+    ).toBeInTheDocument()
+    expect(screen.getByText("Unvollständige Uhrzeit.")).toBeVisible()
+  })
+
+  it("should show error when scheduling date_time is not in the future", async () => {
+    mockDocUnitStore()
+    render(ScheduledPublishingDateTime, {
+      props: { isPublishable: true },
+    })
+
+    const dateField = screen.getByLabelText("Terminiertes Datum")
+    await fireEvent.update(dateField, "01.01.2024")
+
+    expect(dateField).toHaveValue("01.01.2024")
+
+    expect(screen.getByLabelText("Termin setzen")).toBeDisabled()
+
+    expect(
+      screen.getByTestId("scheduledPublishingDate_errors"),
+    ).toBeInTheDocument()
+    expect(
+      screen.getByText(
+        "Der Terminierungszeitpunkt muss in der Zukunft liegen.",
+      ),
+    ).toBeVisible()
+  })
+
+  it("should show error if scheduled date is set and doc unit is not publishable", () => {
+    mockDocUnitStore("2080-10-11T03:00:00.000Z")
+    render(ScheduledPublishingDateTime, {
+      props: { isPublishable: false },
+    })
+
+    expect(screen.getByLabelText("Termin löschen")).toBeEnabled()
+
+    expect(
+      screen.getByTestId("scheduledPublishingDate_errors"),
+    ).toBeInTheDocument()
+    expect(
+      screen.getByText(
+        "Die terminierte Abgabe wird aufgrund von Fehlern in der Plausibilitätsprüfung nicht durchgeführt werden.",
```

Comment:
Sounds a bit weird?
What about
```suggestion
        "Die terminierte Abgabe kann aufgrund von Fehlern in der Plausibilitätsprüfung nicht durchgeführt werden.",
```